terraform/app_service.tf

```diff
@@ -96,6 +97,7 @@ resource "azurerm_linux_web_app" "main" {
     "AUTH_PROVIDER_SCOPE"                                  = "${local.secret_prefix}auth-provider-scope)"
     "AUTH_PROVIDER_CLAIM"                                  = "${local.secret_prefix}auth-provider-claim)"
     "MST_OAUTH_VERIFIER_NAME"                              = "${local.secret_prefix}mst-oauth-verifier-name)"
+    "MST_VETERAN_VERIFIER_NAME"                            = "${local.secret_prefix}mst-veteran-verifier-name)"
```

Comment:
@thekaveman @angela-tran This is all I did in this `app_service.tf` file. Am I supposed to configure something manually in KeyVault after this PR is merged to define some test values.. or something?

Reply:
Yeah, you can configure the value in Key Vault even before this PR is merged. You'll create a new secret called `mst-veteran-verifier-name` with a value similar to the names for other EligibilityVerifiers.

Reply:
Added `mst-veteran-verifier-name` to Key Vault (Dev).